src/tokens/TokenBalances.tsx

```diff
@@ -28,14 +37,18 @@ function TokenBalancesScreen({ navigation }: Props) {
   const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
   const totalBalance = useSelector(totalTokenBalanceSelector)
   const showPriceChangeIndicatorInBalances = useSelector(showPriceChangeIndicatorInBalancesSelector)
+  const shouldVisualizeNFTsInHomeAssetsPage = useSelector(
+    visualizeNFTsEnabledInHomeAssetsPageSelector
+  )
+  const walletAddress = useSelector(walletAddressSelector)
 
   const header = () => {
     return (
       <View style={styles.header}>
         <Text style={fontStyles.navigationHeader}>{t('balances')}</Text>
         <Text style={styles.subtext}>
           {localCurrencySymbol}
-          {totalBalance?.toFormat(2)}
+          {totalBalance?.toFormat(2)} {t('total')}
```

Comment:
hum, i do question whether having "total" here is misleading because we actually don't track all of the tokens a user holds (we only track prices for the tokens that we support) but if Nitya confirms that it should be there then it's okay.

for translations though, i think it's better to add a new key that contains the value so that the order can be decided per language (e.g. in english it might make sense for "$1 total" but for other languages it may need to be "total $1") - for this i'd recommend doing something like `totalBalance ?? t('someNewKey',  totalBalance.toFormat(2)` :) this also has the benefit of if we ever want to remove the "total" text then we are able to push this change from the over-the-air translations